package.json

```diff
@@ -46,15 +46,15 @@
     "prettier": "^2.0.5",
     "rimraf": "^3.0.0",
     "rollup": "^1.27.5",
-    "rollup-plugin-svelte": "^5.0.3",
+    "rollup-plugin-svelte": "^6.1.1",
```

Comment:
Refer this https://github.com/sveltejs/svelte/issues/5665

The package is not building for me in the local environment. If it's not needed I will revert this.

Reply:
@taylorjdawson 
Maybe I am doing something wrong? All I did is clone, `npm install` and `npm build`. Do I have to configure something else?

Reply:
@koushikmohan1996  What node and npm version are you using?

Reply:
@taylorjdawson  npm - 6.14.5, node - 12.18.2

Reply:
Hmm Also not sure if this would make a difference but have you tried building with yarn?

Reply:
@taylorjdawson Weird. That actually fixed the problem. I will revert that. You can review the rest of the changes

Reply:
Nice!